src/test_fixture/sharing.rs

```diff
@@ -2,13 +2,55 @@ use std::iter::zip;
 
 use crate::ff::Field;
 use crate::secret_sharing::{MaliciousReplicated, Replicated};
+use rand::thread_rng;
 use rand::{
     distributions::{Distribution, Standard},
     Rng, RngCore,
 };
 
 use super::{MaliciousShares, ReplicatedShares};
 
+pub trait IntoShares<S> {
+    fn share(self) -> [S; 3];
```

Comment:
I think it would be better to inject `rng` here. I am still hopeful that we will setup property based testing for IPA and it would be very useful to generate a seed and set up PRNG with with. If that test fails, we could replay it because seed value will be saved: https://altsysrq.github.io/proptest-book/proptest/failure-persistence.html